CRM/Queue/BAO/QueueItem.php

```diff
@@ -25,25 +25,46 @@ class CRM_Queue_BAO_QueueItem extends CRM_Queue_DAO_QueueItem {
   /**
    * Ensure that the required SQL table exists.
    *
+   * The `civicrm_queue_item` table is a special requirement - without it, the upgrader cannot run.
+   * The upgrader will make a special request for `findCreateTable()` before computing upgrade-tasks.
+   *
    * @return bool
    *   TRUE if table now exists
    */
-  public static function findCreateTable() {
```

Comment:
We need a step to say `ALTER TABLE civicrm_queue_item ... ADD COLUMN run_count ...`. However, this table is a hard pre-req for the upgrader (since it puts upgrade-tasks into this table). 

There was already a special call-path for twiddling this table (from early on - when `civicrm_queue_item` was first defined). I've just tweaked that call-path to also handle the `ALTER TABLE ... ADD COLUMN ...`